<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WINDOWS-5FHOE7R" name="EmailSubscriptionTest_Edge" tests="2" failures="2" timestamp="8 May 2016 23:58:40 GMT" time="3.881" errors="0">
  <testcase name="testEmailSubscription" time="0.013" classname="com.selenium.hybrid.testcases.EmailSubscriptionTest">
    <failure type="java.lang.AssertionError" message="Fail-Error opening Remote Browser expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Fail-Error opening Remote Browser expected [true] but found [false]
	at com.selenium.hybrid.testcases.EmailSubscriptionTest.testEmailSubscription(EmailSubscriptionTest.java:47)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- testEmailSubscription -->
  <testcase name="testEmailSubscription" time="0.008" classname="com.selenium.hybrid.testcases.EmailSubscriptionTest">
    <failure type="java.lang.AssertionError" message="Fail-Error opening Remote Browser expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Fail-Error opening Remote Browser expected [true] but found [false]
	at com.selenium.hybrid.testcases.EmailSubscriptionTest.testEmailSubscription(EmailSubscriptionTest.java:47)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- testEmailSubscription -->
</testsuite> <!-- EmailSubscriptionTest_Edge -->
